CoreFarms.cs

```diff
@@ -2420,7 +2420,7 @@ public void SwagTokenA(int quant = 100)
         {
             Core.FarmingLogger($"Super-Fan Swag Token A", quant);
 
-            Core.RegisterQuests(1310);
+            Core.RegisterQuests(1310,1312,1313,1314);
```

Comment:
do all the quest requires for these drop for that map? if so ty.

Reply:
yes ofc